requirements.txt

```diff
@@ -1,6 +1,3 @@
-mlflow[pipelines]>=1.27.0
-pandas>=1.3.*
-scikit-learn>=1.1.*
```

Comment:
I thought we needed `scikit-learn` for regression template. Let me know if I am missing something or is that included as part of mlflow default?

Reply:
@sunishsheth2009 MLflow 2.0 includes scikit-learn as a dependency: https://github.com/mlflow/mlflow/blob/ff9a05c7abe68c74ff5563f27c1b5956a4e7ad63/requirements/core-requirements.txt#L15

Reply:
Got it thanks :)